ena-submission/src/ena_deposition/create_sample.py

```diff
@@ -410,33 +409,42 @@ def sample_table_handle_errors(
     db_config: SimpleConnectionPool,
     config: Config,
     slack_config: SlackConfig,
-    time_threshold: int = 15,
-    slack_time_threshold: int = 12,
-):
+    last_retry_time: datetime | None,
+    submitting_time_threshold_min: int = 15,
```

Comment:
Why are we not using config for these settings? Saves on line count and also makes it configurable!